config/env/test.js

```diff
@@ -12,6 +12,15 @@ module.exports = {
     // Enable mongoose debug mode
     debug: process.env.MONGODB_DEBUG || false
   },
+  log: {
+    // Can specify one of 'combined', 'common', 'dev', 'short', 'tiny'
+    format: process.env.LOG_FORMAT || 'combined',
+    // Stream defaults to process.stdout
+    // Uncomment to enable logging to a log on the file system
```

Comment:
It was just a left over comment from when I copied the settings from the development config. 

I can remove it, but it also does make sense to leave it?